### PR TITLE
chore: add download links for all samples

### PR DESCRIPTION
**Pull request by andygup**
This PR reverts the changes made in https://github.com/Esri/jsapi-resources/pull/518/files. 

This update only allows for downloading individual samples. There currently isn't a download-all for any of the sample directories.

### CODE REVIEW

**Review by sagewall — APPROVED** [review]

File: component-samples/coding-components/samples/angular/README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Coding components Angular sample
 
+📁 **[Click here to download this directory as a ZIP file](https://github.com/Esri/jsapi-resources/blob/main/zips/coding-components-sample-angular.zip)** 📁
```
Comment (omarkawach) [inline component-samples/coding-components/samples/angular/README.md:3]: @manugadde all coding-component sample READMEs will need similar fixes

```suggestion
📁 **[Click here to download this directory as a ZIP file](https://esri.github.io/jsapi-resources/zips/coding-components-sample-angular.zip)** 📁
```

File: component-samples/map-components/samples/angular/README.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # Map components Angular sample
 
+📁 **[Click here to download this directory as a ZIP file](https://github.com/Esri/jsapi-resources/blob/main/zips/map-component-sample-angular.zip)** 📁
+
+
```
Comment (omarkawach) [inline component-samples/map-components/samples/angular/README.md:5]:
```suggestion
📁 **[Click here to download this directory as a ZIP file](https://github.com/Esri/jsapi-resources/blob/main/zips/map-component-sample-angular.zip)** 📁


```

File: component-samples/map-components/samples/angular/README.md [hunk 3]
```diff
@@ -1,5 +1,8 @@
 # Map components Angular sample
 
+📁 **[Click here to download this directory as a ZIP file](https://github.com/Esri/jsapi-resources/blob/main/zips/map-component-sample-angular.zip)** 📁
```
Comment (omarkawach) [inline component-samples/map-components/samples/angular/README.md:3]: @manugadde all map-component sample READMEs will need to be updated

```suggestion
📁 **[Click here to download this directory as a ZIP file](https://esri.github.io/jsapi-resources/zips/map-component-sample-angular.zip)** 📁
```

File: core-samples/README.md [hunk 4]
```diff
@@ -2,6 +2,24 @@
...
+* [Deno](https://esri.github.io/jsapi-resources/zips/core-sample-jsapi-deno.zip)
+* [esbuild (no framework)](https://esri.github.io/jsapi-resources/zips/core-sample-esbuild.zip)
+* [Node.js](https://esri.github.io/jsapi-resources/zips/core-sample-jsapi-node.zip)
+* [OAuth](https://esri.github.io/jsapi-resources/zips/core-sample-jsapi-oauth.zip)
+* [React/Vite](https://esri.github.io/jsapi-resources/zips/core-sample-jsapi-react.zip)
+* [Rollup.js (no framework)](https://esri.github.io/jsapi-resources/zips/core-sample-rollup.zip)
+* [Vue.js/Vite](https://esri.github.io/jsapi-resources/zips/core-sample-jsapi-vue.zip)
+* [Vite/TypeScript](https://esri.github.io/jsapi-resources/zips/core-sample-jsapi-vite-ts.zip)
+* [Webpack (no framework)](https://esri.github.io/jsapi-resources/zips/core-sample-webpack.zip)
+
```
Comment (omarkawach) [inline core-samples/README.md:22]: @jimmychen97 can we get something similar for /charts-components README

@manugadde same for coding-components and map-components for convenience

File: core-samples/README.md [hunk 5]
```diff
@@ -2,6 +2,24 @@
 
 The samples in this directory demonstrate using [@arcgis/core](https://www.npmjs.com/package/@arcgis/core) ES modules with various frameworks, module bundlers and build tools. `@arcgis/core` is installed using [npm](https://docs.npmjs.com/downloading-and-installing-packages-locally), and is intended for use in local builds that you host.
 
+## Download
+
+You can use these links to download all of the samples, or individual samples:
```
Comment (omarkawach) [inline core-samples/README.md:7]: @andygup we don't have a way yet to allow for downloading all samples, right?

```suggestion
You can use these links to download individual samples:
```

**Review by omarkawach — CHANGES_REQUESTED** [review]

File: core-samples/README.md (hunk 5 above)
Comment (andygup) [reply core-samples/README.md:7]: Oops, I missed that bit of wording.

File: component-samples/map-components/samples/angular/README.md [hunk 6]
```diff
@@ -1,5 +1,8 @@
 # Map components Angular sample
 
+📁 **[Click here to download this directory as a ZIP file](https://esri.github.io/jsapi-resources/zips/coding-components-sample-angular.zip)** 📁
```
Comment (omarkawach) [inline component-samples/map-components/samples/angular/README.md:3]: @manugadde the links are mixed up for map and coding components.

```suggestion
📁 **[Click here to download this directory as a ZIP file](https://esri.github.io/jsapi-resources/zips/map-component-sample-angular.zip)** 📁
```

**Review by omarkawach — CHANGES_REQUESTED** [review]
@manugadde

**Review by manugadde — APPROVED** [review]

**Review by omarkawach — APPROVED** [review]

**Pull request merged**